app/models/microcosm_member.rb

```diff
@@ -0,0 +1,30 @@
+# == Schema Information
+#
+# Table name: microcosm_members
+#
+#  id           :bigint(8)        not null, primary key
+#  microcosm_id :integer          not null
+#  user_id      :integer          not null
+#  role         :string(64)       not null
```

Comment:
Could be an enum

Reply:
Created https://github.com/openbrian/osm-microcosms/issues/9 to track this.